Three/include/CGAL/Three/Viewer_interface.h

```diff
@@ -136,6 +136,7 @@ class VIEWER_EXPORT Viewer_interface : public QGLViewer, public QOpenGLFunctions
   void requestContextMenu(QPoint global_pos);
   //!Defined automatically in moc.
   void selectedPoint(double, double, double);
+  void pointSelected(QPoint p);
   //!Defined automatically in moc.
```

Comment:
By the way, the base class `QGLViewer` already has:
```
void pointSelected 	( 	const QMouseEvent *  	e	)
```
that does almost the same thing: the `QPoint p` is `e->pos()`.

I really would prefer that the `.h` of the framework are only modified when necessary, because that may mean:
  - recompilations
  - or ABI-incompatibilities.